src/protocols/at-ps2/mouse_interface.c

```
@@ -76,28 +78,28 @@ typedef enum { X_POS, Y_POS, Z_POS } mousepos_index;
- * If the sign bit is set and the position is non-zero, the position is decremented by 0x100.
+ * If the sign bit is set, the position is decremented by 256 for proper 9-bit signed conversion.
  * The resulting position is then clamped between -127 and 127.
  *
- * @param pos      The position value.
- * @param sign_bit The sign bit.
+ * @param pos      The position value (8-bit magnitude).
+ * @param sign_bit The sign bit (9th bit for signed interpretation).
  *
  * @return The calculated XY movement as an int8_t value.
  */
 int8_t get_xy_movement(uint8_t pos, int sign_bit) {
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'get_xy_movement' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)